src/js/utils/request.js

```diff
@@ -0,0 +1,11 @@
+export const request = async ({ uri, method, type, body, headers }) => {
+  const response = await fetch(uri, { method, body, headers }).then((response) => {
```

Comment:
```suggestion
export const request = async ({ uri, body, ...rest }) => {
  const response = await fetch(uri, { ...rest }).then((response) => {
```

rest parameter와 object spread syntax를 잘 사용하면 기존의 인터페이스를 확장하듯이 작성할 수 있습니다 👍

Reply:
이부분은 로직을 바꾸게 되어서 rest parameter와 object spread syntax를 사용해보지 못했지만, 다음에 기회가 되면 꼭 사용해보고싶네요! 🧚🏼‍♀️🧚🏼‍♂️